appdev/guides/importing-CH-urldispatcher.rst

```diff
@@ -0,0 +1,157 @@
+
+Also, a URI formatted geo:xxx,xxx should open in uNav, as it’s defined in line 11.
+
+And how do we manage the received URL?
+--------------------------------------
+
+After the `URLdispatcher` sends the link to the correspondent app, we need to handle that URL or URI in the targeted app. Let’s see how to do that:
+
+In the main qml file, we need to add some code to know what to do with the dispatched URL. First add an `Arguments element <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L189>`_ that holds the URL, as is done, for example, in the `Linphone app <https://open-store.io/app/linphone.cibersheep>`_. Also, we add `connection to the URI Handler <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L200>`_ with a Connection element with `UriHandler` as a target.
+
+.. code:: qml
+
+        Arguments {
+            id: args
+        
+            Argument {
+                name: 'url'
+                help: i18n.tr('Incoming Call from URL')
+                required: false
+                valueNames: ['URL']
+            }
+        }
+
+        Connections {
+            target: UriHandler
+        
+            onOpened: {
+                console.log('Open from UriHandler')
+        
+                if (uris.length > 0) {
+                    console.log('Incoming call from UriHandler ' + uris[0]);
+                    showIncomingCall(uris[0]);
+                }
+            }
+        }
+
+This code will manage a URI in the form `linphone://sip:xxx@xxx.xx` when the app is opened. But what do we need to do to handle this link when the app is closed?
+
+We need to add a little bit `extra code <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L69>`_ that will cover two cases:
+1) We receive one URL
+2) We receive more than one URL
+
+.. code:: qml
+
+        Component.onCompleted: {
+            //Check if opened the app because we have an incoming call
+            if (args.values.url && args.values.url.match(/^linphone/)) {
+
+                console.log("Incoming Call on Closed App")
+                showIncomingCall(args.values.url);
+
+            } else if (Qt.application.arguments && Qt.application.arguments.length > 0) {
+
+                for (var i = 0; i < Qt.application.arguments.length; i++) {
+                    if (Qt.application.arguments[i].match(/^linphone/)) {
+                        showIncomingCall(Qt.application.arguments[i]);
+                    }
+                }
+            }
+
+            //Start timer for Registering Status
+            checkStatus.start()
+        }
+
+
+All well and good but what happens if more than one app has the same URL type defined?
+--------------------------------------------------------------------------------------
```

Comment:
I'd shorten this to "What happens if more than one app has the same URL type defined?"